### PR TITLE
OSDOCS-3546: Egress firewall supports logging

### PR DESCRIPTION
**Pull request by jboxman-rh**
Because the logging mechanism for EgressFirewall is an extension
of that for network policy, this PR reshuffles and expands the
content to reflect this. The new location of the content is with
the rest of the OVN-Kubernetes documentation.

- https://issues.redhat.com/browse/OSDOCS-3546

Preview: [Logging for egress firewall and network policy rules](https://49778--docspreview.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/logging-network-policy.html)

Version(s): 4.12

### CODE REVIEW

**Comment by jboxman-rh** [comment]
@andreaskaris, this PR updates the documentation regarding network policy audits so that it also applies to egress firewall.

What do you think?

Thanks!

**Comment by andreaskaris** [comment]
a) There's a typo here in the preview https://49778--docspreview.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/logging-network-policy.html
~~~
annotate --overwrite namespace <namespace> k8s.ovn.org/acl-logging={}
~~~
... it shoud be `oc annotate ...`

b) Also, I know it's been in our doc for ages now, but I don't understand why we said the above and not simply removed the annotation:
~~~
oc annotate --overwrite namespace <namespace> k8s.ovn.org/acl-logging-
~~~

That would IMO be cleaner (plus the `={}` actually had a bug until recently https://bugzilla.redhat.com/show_bug.cgi?id=2111362)

**Comment by jboxman-rh** [comment]
@andreaskaris, looks like the missing `oc` was fixed here: db2790bfb99ec6b217f219afc5fc9e7190132aba

**Comment by jboxman-rh** [comment]
@andreaskaris, I've created a separate PR to fix the annotation removal across all versions it applies: https://github.com/openshift/openshift-docs/pull/50401

**Comment by jboxman-rh** [comment]
@andreaskaris, any there any other updates that you'd suggest? Thanks!

**Comment by andreaskaris** [comment]
@jboxman-rh Is the link in the summary still the latest preview version, or how can I get that?

**Comment by jboxman-rh** [comment]
> @jboxman-rh Is the link in the summary still the latest preview version, or how can I get that?

It is, yes. It updates after some delay automatically.

File: logging/cluster-logging-external.adoc [hunk 1]
```diff
@@ -199,6 +199,6 @@ include::modules/cluster-logging-collector-collecting-ovn-logs.adoc[leveloffset=
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../networking/network_policy/logging-network-policy.adoc#nw-networkpolicy-audit-concept_logging-network-policy[Network policy audit logging]
+* xref:../networking/ovn_kubernetes_network_provider/logging-network-policy.adoc#logging-network-policy[Logging network policy events]
```
Comment (andreaskaris) [inline logging/cluster-logging-external.adoc:202]: Is this here correct, or should it contain references to Egress FW as well?

**Comment by jboxman-rh** [comment]
@huiran0826, can you take a look when you have time? Thanks!

File: networking/ovn_kubernetes_network_provider/logging-network-policy.adoc [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
 [id="logging-network-policy"]
-= Logging network policy events
+= Logging for egress firewall and network policy rules
 include::_attributes/common-attributes.adoc[]
 :context: logging-network-policy
 
 toc::[]
 
-As a cluster administrator, you can configure network policy audit logging for your cluster and enable logging for one or more namespaces.
+As a cluster administrator, you can configure  audit logging for your cluster and enable logging for one or more namespaces. {product-title} produces audit logs for both egress firewalls and network policies.
```
Comment (mburke5678) [inline networking/ovn_kubernetes_network_provider/logging-network-policy.adoc:9]: Is there an extra space here?

File: networking/ovn_kubernetes_network_provider/logging-network-policy.adoc [hunk 3]
```diff
@@ -1,23 +1,24 @@
...
 ====
 
 include::modules/nw-networkpolicy-audit-concept.adoc[leveloffset=+1]
 
-== Network policy audit configuration
+[id="network-policy-audit-configuration-{context}"]
+== Audit configuration
 
-The configuration for audit logging is specified as part of the OVN-Kubernetes cluster network provider configuration. The following YAML illustrates default values for network policy audit logging feature.
+The configuration for audit logging is specified as part of the OVN-Kubernetes cluster network provider configuration. The following YAML illustrates default values for the audit logging feature.
```
Comment (mburke5678) [inline networking/ovn_kubernetes_network_provider/logging-network-policy.adoc:21]: s/the audit logging feature/audit logging

I think this is the only place where it is called _the audit logging feature_.

**Comment by mburke5678** [comment]
@jboxman-rh A couple of nits. Otherwise LGTM

**Pull request merged**